### PR TITLE
Added all files for Fres-98 design

### PR DESCRIPTION
**Pull request by InnoraftxAmarta**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+        <div class="banner-flower-div">
+          <img src="image/flower.png" class="banner-flower">
+          <h2 class="flowernum">01</h2>
+        </div>
+        
+    </div>
+    <!-- banner ends here  -->
+    <!-- shopping starts here div2 -->
+    <div class="shopping">
+      <img src="image/brush.png" class="l-brush">
```
Comment (yogesh-innoraft) [inline index.html]: add alt tag in all images and pass related text in it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+              </ul>
+            </div>
+          </div>
+          <p class="copyright-para">
+            Copyright 2021 Fashion with Ethics All Rights Reserved.
+          </p>
+        </div>
+      </div>
+    </footer>
+    <!-- <script
```
Comment (yogesh-innoraft) [inline index.html]: remove commented code

File: scss/style.scss [hunk 3]
```diff
@@ -0,0 +1,930 @@
+@import url("https://fonts.googleapis.com/css2?family=Dela+Gothic+One&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Chivo:ital,wght@1,700&display=swap");
```
Comment (yogesh-innoraft) [inline scss/style.scss:2]: why are you adding fonts in HTML and CSS both files?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- bootstrap css  -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <!-- google fonts css  -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Chivo:ital,wght@1,700&display=swap" rel="stylesheet">
+    <!-- bootstrap cdn  -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css">
```
Comment (yogesh-innoraft) [inline index.html:11]: is it needed?

File: scss/style.scss (hunk 3 above)
Comment (InnoraftxAmarta) [reply scss/style.scss:2]: removed

File: index.html (hunk 2 above)
Comment (InnoraftxAmarta) [reply index.html]: removed

File: index.html (hunk 1 above)
Comment (InnoraftxAmarta) [reply index.html]: added alt text

File: index.html (hunk 4 above)
Comment (InnoraftxAmarta) [reply index.html:11]: no i just removed that

**Review by yogesh-innoraft — APPROVED** [review]

**Pull request merged**